packages/flutter_tools/test/general.shard/plugins_test.dart

```diff
@@ -1414,6 +1416,42 @@ flutter:
       });
 
     });
+
+    testWithoutContext('Symlink failures give developer mode instructions on recent versions of Windows', () async {
+      final MockPlatform platform = MockPlatform();
```

Comment:
Instead of using the mock use `FakePlatform(operatingSystem: 'windows')`

Reply:
Done.